visuals/MDS.py

```diff
@@ -0,0 +1,74 @@
+from votekit.profile import PreferenceProfile
+from metrics.distances import earth_mover_dist, Lp_dist
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn import manifold
+import sys
+
+sys.path.append("../")
```

Comment:
this should not be in this file, importing a module should not change `sys.path` since a lot of weird things can happen if you do

Reply:
done